index.html

```diff
@@ -953,7 +954,7 @@ <h2>Methods</h2>
                 slots to <code>false</code>.</li>
                 <li>Set <var>stream</var>'s <a>[[\QuicStreamState]]</a> slot to
                 <code>closing</code>.</li>
-                <li> If the <var>stream</var> has received a STREAM frame with the FIN bit set,
+                <li>If <var>stream</var> has <a>finished reading</a>,

```

Comment:
Does this make sense? t seems to me that if you've received a FIN bit at this point you should transition to close, and it shouldn't depend on finishing reading. You can't read after this point anyways (reset() changes [[Readable]] to false). 

This also contradicts 2. in the closed definition.

Reply:
Reverted the change in line 956/957.